docs/runbooks/LowVirtControllersCount.md

```diff
@@ -2,42 +2,46 @@
 
 ## Meaning
 
-More than one virt-controller pod should be ready to ensure high-availability. The current default number of replicas is 2.
+This alert fires when a low number of `virt-controller` devices is detected. More than one `virt-controller` pod should be available to ensure high availability. The current default number of replicas is 2.
```

Comment:
I'm wondering whether this wording might be clearer.

"This alert fires when a low number of `virt-controller` pods is detected. At least one `virt-controller` pod must be available in order to ensure high availability. The default number of replicas is `2`.